### PR TITLE
Adding DLA Dao Token to Cosmos/Juno Registry 

### PR DESCRIPTION
**Pull request by frameworkfortune**

### CODE REVIEW

File: juno/assetlist.json [hunk 1]
```diff
@@ -251,6 +251,32 @@
...
+            "denom": "dla",
+            "exponent": 6
+          }
+        ],
+        "base": "juno1sfwye65qxcfsc837gu5qcprcz7w49gkv3wnat04764ld76hy3arqs779tr",
+        "name": "Digital Land Acquisition DAO",
+        "display": "DLA DAO",
+        "symbol": "DLA",
+        "logo_URIs": {
+          "png": "https://github.com/frameworkfortune/chain-registry/blob/master/juno/images/dla.png?raw=true",
```
Comment (JeremyParish69) [inline juno/assetlist.json:274]: can we please change the URI to: https://github.com/cosmos/chain-registry/blob/master/juno/images/dla.png

File: juno/assetlist.json [hunk 2]
```diff
@@ -251,6 +251,32 @@
...
+          "svg": "https://raw.githubusercontent.com/frameworkfortune/chain-registry/efd01f073d7c1ed7b875ebce30209edb421b39bd/juno/images/dla.svg"
```
Comment (JeremyParish69) [inline juno/assetlist.json:275]: can we please change the URI to: https://github.com/cosmos/chain-registry/blob/master/juno/images/dla.svg

Also, why are the two images different? Is one version preferred? or is it a rendering issue?

File: juno/assetlist.json (hunk 1 above)
Comment (frameworkfortune) [reply juno/assetlist.json:274]: Yes

File: juno/assetlist.json (hunk 2 above)
Comment (frameworkfortune) [reply juno/assetlist.json:275]: The dla.png is the correct the graphic. Not sure why the other one looks like that unless it is a rendering issue? Is there a specific size for the svg file it needs to be?

File: juno/assetlist.json [hunk 3]
```diff
@@ -251,6 +251,32 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "dla",
+            "exponent": 6
+          }
+        ],
+        "base": "juno1sfwye65qxcfsc837gu5qcprcz7w49gkv3wnat04764ld76hy3arqs779tr",
+        "name": "Digital Land Acquisition DAO",
+        "display": "DLA DAO",
```
Comment (JeremyParish69) [inline juno/assetlist.json:271]: This must be one of the listed denominations. I assume it should be "dla"

**Review by frameworkfortune — COMMENTED** [review]
Yes thank you Jeremy for all the help with the merge! I tipped you through brave but it said you haven't set up your wallet yet! Might want to setup your brave browser wallet encase other people have been tipping you as well!

**Pull request merged**